transcom/mymove
# B-18342 Fix for DDDSIT sit departure date formatting with UTC

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/TeamRoom.mvc/Show/808731)

## Summary

For a DDDSIT service item, the SIT departure date was showing up as a mismatch from what the updated value should be and what was in the database.

Issue shown:
 Updated value is 2/2/20 but the value on the details section is 2/1/20.
![image](https://github.com/transcom/mymove/assets/147537467/baed65de-9690-4808-ba71-944ac1811ec1)
![image](https://github.com/transcom/mymove/assets/147537467/145e25bf-d4e4-44ed-9f4c-a86d65d69dcb)

The fix is just using a format date function that accounts for UTC. With the fix, this is now what appears:
![image](https://github.com/transcom/mymove/assets/147537467/9f16ede4-a9c4-450a-a879-a98b6b476803)


## Verification Steps for Reviewers

These are to be checked by a reviewer.

1. Resubmit a DDDSIT and enter a SIT departure date. 
2. Verify that the date matches the updated SIT departure date for the service item details, the tooltip that shows what changed, and the database also matches the updated value.


### How to test

1. Access the office application.
4. Login as a TOO.
5. Verify SIT departure date in the service item details is correct and matches the database value.

### Frontend
